test/parallel/test-tls-env-extra-ca-file-load.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+// Flags: --expose-internals
+
+const common = require('../common');
+
+if (!common.hasCrypto)
+  common.skip('missing crypto');
+
+const assert = require('assert');
+const tls = require('tls');
+const fixtures = require('../common/fixtures');
+const { internalBinding } = require('internal/test/binding');
+const binding = internalBinding('crypto');
+
+const { fork } = require('child_process');
+
+// This test ensures that extra certificates are loaded at startup.
+{
+  if (process.env.CHILD_USE_EXTRA_CA_CERTS === 'yes') {
+    assert.strictEqual(binding.isExtraRootCertsFileLoaded(), true);
+    return;
+  }
+
+  if (process.env.CHILD_USE_EXTRA_CA_CERTS === 'no') {
```

Comment:
IMHO this will be more readable is it was:
```js
if {
} else if {
} else if {
} else {
  assert.fail("Bad child process invocation")
}
```
Having a `if (process.argv[2] !== 'child')` guard remove the need to do `return` in each clause